modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -104,6 +120,13 @@ template<typename Net> class Params {
               , 1u} {
     };
 
+    /** @overload
+    This constructor for pre-compiled networks. Model is imported from pre-compiled
+    blob.
+
+    @param model Path to model.
+    @param device Name of device.
```

Comment:
Please unify the `device` description among the constructors (use one proposed from `ParamDesc`).